src/imitation/policies/serialize.py

```diff
@@ -85,17 +85,21 @@ def _load_zero(path: str, env: gym.Env) -> ZeroPolicy:
 
 
 STABLE_BASELINES_CLASSES = {
-    'ppo1': (stable_baselines.PPO1, 'policy_pi'),
-    'ppo2': (stable_baselines.PPO2, 'act_model'),
+    'ppo1': ('stable_baselines:PPO1', 'policy_pi'),
+    'ppo2': ('stable_baselines:PPO2', 'act_model'),
 }
 
 
 AGENT_LOADERS: Dict[str, PolicyLoaderFn] = {
     'random': _load_random,
     'zero': _load_zero,
 }
-for k, (cls, attr) in STABLE_BASELINES_CLASSES.items():
-  AGENT_LOADERS[k] = _load_stable_baselines(cls, attr)
+for k, (cls_name, attr) in STABLE_BASELINES_CLASSES.items():
+  try:
+    cls = registry.load_attr(cls_name)
+    AGENT_LOADERS[k] = _load_stable_baselines(cls, attr)
+  except (AttributeError, ImportError):
```

Comment:
Consider adding comment to explain we have optional dependencies so this is an expected/non-critical error